cfgen/cfgen

```diff
@@ -892,17 +907,17 @@ class LibfabricEndpoint(Endpoint):
     ep_map: Dict[Tuple[str, int], int] = {}
 
     def __init__(self, NetFamily: str, proto: NetProtocol, ipaddr: str,
-                 portalgroup: LibfabPortalGroup):
+                 portalgroup: int):
         self.netfamily = NetFamily
         self.proto = proto
         assert ipaddr
         self.ipaddr = ipaddr
-        assert portalgroup.value > 0
+        assert portalgroup > 0
         self.portalgroup = portalgroup
         self.portal = 0
-        self.portal = self.ep_map.setdefault((ipaddr, portalgroup.value),
-                                             (portalgroup.value + 1))
-        self.ep_map[(ipaddr, portalgroup.value)] += 1
+        self.portal = self.ep_map.setdefault((ipaddr, portalgroup),
+                                             (portalgroup + 1))
```

Comment:
and corresponding change here as well, `portalgroup.port_next(proct)` and similar changes else where.